groovy/postInit/mod/MachineRecipes.groovy

```diff
@@ -944,4 +953,39 @@ recipemap('assembler').recipeBuilder()
 		.outputs(metaitem('power_unit.iv'))
 		.EUt(7680)
 		.duration(150)
-		.buildAndRegister()
\ No newline at end of file
+		.buildAndRegister()
+
+//New SuSy blocks
```

Comment:
All the stuff below this point has to be removed, the recipes are already dealt with on another PR.
Also still using metaitems in recipe inputs.